src/main/java/uk/gov/hmcts/reform/sscs/config/SwaggerConfiguration.java

```diff
@@ -22,5 +20,19 @@ public Docket api() {
             .paths(PathSelectors.any())
             .build();
     }
+    */
+
+    // This is boilerplate and requires implementing real values of the above commented code
+    @Bean
+    public OpenAPI api() {
```

Comment:
Could we add values in here, for title/description, etc? Any reason why we're boilerplating it?

Reply:
This is just as far as I got with following instructions from here https://springdoc.org/migrating-from-springfox.html

I can't figure out how the original version should be migrated over to the new one with those details.

Reply:
Right, maybe some more descriptive boilerplate then, like "SSCS Evidence Share" for title, description cribbed from the README.md file, something like that.

Reply:
How'd I do?

Reply:
Good, only thing I'd change is removing the external docs pointing to our github repo, as it's under license. If you can add a .url to the Info() object however, do that.

Reply:
changed